typeDefs.js

```diff
@@ -396,6 +405,7 @@ const typeDefs = gql`
     switchUserStatus(id: ID!): LogicalResult
     resetPassword(password: String!, token: String!): Boolean
     checkIfTokenIsValid(token: String!): Boolean
+    completeAdminRegister(user: SpecialUserConfirmInput!,token: String!): LogicalResult
```

Comment:
Please, make `LogicalResult` not-nullable

Reply:
fixed